doc/troubleshooting.md

```diff
@@ -0,0 +1,362 @@
+# Troubleshooting compliance-operator
+This document describes how to troubleshoot problems with compliance
+operator. The information can be useful either to diagnose a problem or
+provide information in a bug report.
+
+Please refer to the README for general information about the API objects.
+
+## General tips
+
+   * The Compliance Operator emits Kubernetes events when something
+     important happens. You can either view all events in the cluster (`oc get events
+     -nopenshift-compliance`) or events for an object, e.g. for a scan
+     (`oc get compliancescan/$scan`)
+
+   * The Compliance Operator consists of several controllers, roughly
+     one per API object. It could be handy to filter only those controller that pertain to
+     the API object having issues, e.g. if a `ComplianceRemediation` can't be applied,
+     the first place to look might be messages from the `remediationctrl` controller. You can filter the messages
+     from a single controller e.g. using `jq`:
+     `oc logs compliance-operator-775d7bddbd-gj58f | jq -c 'select(.logger == "controller_profilebundle")' `
```

Comment:
It would be `profilebundlectrl` nowadays

Reply:
fixed